app/client/cli/event.go

```diff
@@ -63,22 +61,20 @@ func (c *Client) CmdEventCreate() *cobra.Command {
 				Capacity:  int32(capacity),
 				Available: int32(available),
 			})
-
 			if err != nil {
 				PrintError(err)
 				return
 			}
-
 			for {
-				_, err := stream.Recv()
+				statusReport, err := stream.Recv()
 				if err == io.EOF {
 					break
 				}
-
 				if err != nil {
 					PrintError(err)
 					return
 				}
+				fmt.Println(statusReport.Status)
```

Comment:
I would suggest to have the progress bar on the client side, rather than on the daemon side. This bar is then incremented every time that a message is retrieved from the stream

Reply:
Yeah, but what will be the difference ? Same procedure will applied again.

Reply:
One difference will be that you will not see this stack of progress bars (like your example gif), but a single one that updates dynamically (which I prefer). I guess it's also a matter of responsibility; the daemon should not really be concerned with how the client displays the information that it sends.